code/sql/schema.sql

```diff
@@ -1,96 +1,121 @@
 CREATE SCHEMA waze;
 
+CREATE TABLE waze.data_files
+(
+"id"                                SERIAL PRIMARY KEY NOT NULL,
+"start_time_millis"                 BIGINT NOT NULL,
+"end_time_millis"                   BIGINT NOT NULL,
+"start_time"                        TIMESTAMP,
+"end_time"                          TIMESTAMP,
+"date_created"                      TIMESTAMP,
+"date_updated"                      TIMESTAMP,
+"file_name"                         TEXT NOT NULL,
+"json_hash"                         uuid NOT NULL,
+UNIQUE ("start_time_millis", "json_hash")
```

Comment:
My plan is to actually embed the start_time_millis into the hash, so we won't need a separate unique constraint

Reply:
Yes agreed.